### PR TITLE
remove logo for unidentified contributors

### LINKED ISSUE #707: ORCID logo being displayed for unidentified contributors
image

The above image would seem to imply that each of the three authors has an associated ORCID, however if you click on the symbol associated with "Inema, Ingrid", it takes you to orcid.org/unidentified. I think it would be better to omit the ORCID symbol for users that do not have an associated ORCID. This would be more in line with what the user expects and will help us identify where we are missing this data for specific contributors.

### PR DESCRIPTION
**Pull request by sanuann**
- closes #707
- removes contributor and keywords rows if no data present

### CODE REVIEW

File: web/src/views/DandisetLandingView/DandisetMain.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
       align="center"
     >
       <v-col cols="auto">
-        ID: {{ meta.id }}
```
Comment (satra) [inline web/src/views/DandisetLandingView/DandisetMain.vue:8]: why is this being removed?

Comment (sanuann) [reply web/src/views/DandisetLandingView/DandisetMain.vue]: sorry by mistake

**Comment by satra** [comment]
this doesn't seem to be working. see dandiset 4 as an example. still several orcid logos pointing to undefined.

File: web/src/views/DandisetLandingView/DandisetMain.vue [hunk 2]
```diff
@@ -180,20 +182,23 @@
...
-          >
-            <v-chip
-              small
-              style="margin: 5px;"
-              class="grey darken-2 font-weight-bold white--text"
-            > {{ key }} </v-chip>
-          </span>
-        </v-col>
-      </v-row>
+<!--      <v-row-->
```
Comment (satra) [inline web/src/views/DandisetLandingView/DandisetMain.vue:185]: did you mean to comment this out?

Comment (sanuann) [reply web/src/views/DandisetLandingView/DandisetMain.vue]: oops, was just testing something!

**Comment by satra** [comment]
one the topic of identifiers, i suspect that name is being processed the same way independent of whether it's a person or organization: 

image

we should not put a `,` in the organization names.

**Comment by sanuann** [comment]
> we should not put a `,` in the organization names.

The `,` appears the names are in the "Last name, First name" format. through the UI "edit metadata" option I don't see a place where you can put the last and first names separately. any idea where that comes from?

**Comment by satra** [comment]
ah no worries then. we need to fix this at the metadata level.

**Comment by satra** [comment]
@waxlamp - this is ready for review. the main issue is handled.

**Review by waxlamp — COMMENTED** [review]
There is more in this PR than just the orcid badge fix--can you explain why the other changes are included?

**Comment by sanuann** [comment]
> There is more in this PR than just the orcid badge fix--can you explain why the other changes are included?

to remove the rows if there is no list of contributors or keywords.

**Comment by waxlamp** [comment]
> > There is more in this PR than just the orcid badge fix--can you explain why the other changes are included?
> 
> to remove the rows if there is no list of contributors or keywords.

Thanks for explaining. In general, it's good to keep PRs scoped as narrowly as possible, to make reviewing easier to and lessen the risk of a bug being introduced in any given PR.

**Review by waxlamp — APPROVED** [review]

**Pull request merged**